openfisca_web_api_preview/loader/tax_benefit_system.py

```diff
@@ -13,9 +14,9 @@ def build_tax_benefit_system(country_package_name):
         country_package = importlib.import_module(country_package_name)
     except ImportError:
         message = linesep.join([traceback.format_exc(),
-                                u'Could not import module `{}`.'.format(country_package_name).encode('utf-8'),
```

Comment:
Wow, weird, how was it working previously? It was joining one line of bytes with lines of str? How was that possible? I must be missing something...

Reply:
I guess `join` is flexible enough to handle both `bytes` and `str`:

`' '.join([u'X', b'Y'])` works in Python 2

Reply:
I tried it in py2, and it didn't work: I believe it's the opposite of your example, and also you need to have a mix of chars that won't auto decode properly, like:

```python
u"\n".join(["é", u"é"])
```

This will fail, but I guess this specific use case never occured in real life. Wow, lucky, and yay for py3 and proper explicit unicode handling!